unix/Makefile

```diff
@@ -153,3 +158,9 @@ coverage_test: coverage
 	cd ../tests && MICROPY_MICROPYTHON=../$(DIRNAME)/micropython_coverage ./run-tests --emit native
 	gcov -o build-coverage/py ../py/*.c
 	gcov -o build-coverage/extmod ../extmod/*.c
+
+_frozen_upip.c: frozen_upip/upip.py
+	../tools/make-frozen.py frozen_upip > $@
+
+frozen_upip/upip.py:
+	../tools/bootstrap_upip.sh frozen_upip
```

Comment:
This is no good, since now the standard unix build requires internet connectivity (and wget).  I'm strongly -1 on that.  See eg Travis failure.